File: frontend/src/lib/google-sheets.ts
/**
 * Inert compatibility marker for the retired browser Google Sheets sink.
 *
 * Delivery is performed by the backend's durable, authenticated outbox. This
 * module deliberately exposes no URL storage, network request, or send API.
 */
export const LEGACY_BROWSER_GOOGLE_SHEETS_SINK_ENABLED = false as const;

File: backend/app/services/integrations/google_sheets.py
"""Inert compatibility marker for the retired direct Google Sheets sink.

Google Sheets delivery is owned by ``google_sheets_mirror`` and its durable
database outbox.  This module intentionally contains no delivery functions,
network client, event handler, database access, or runtime registration.
The path remains present so upgrades and regression guards do not mistake the
retirement of the legacy implementation for an unreviewed source deletion.
"""

LEGACY_DIRECT_SINK_ENABLED = False

__all__ = ["LEGACY_DIRECT_SINK_ENABLED"]
